profiles/winbind/password-auth

```diff
@@ -5,7 +5,7 @@ auth        sufficient                                   pam_u2f.so cue
 auth        required                                     pam_u2f.so cue nouserok                                  {include if "with-pam-u2f-2fa"}
 auth        sufficient                                   pam_unix.so {if not "without-nullok":nullok} try_first_pass
 auth        requisite                                    pam_succeed_if.so uid >= 1000 quiet_success
-auth        sufficient                                   pam_winbind.so {if "with-krb5":krb5_auth} use_first_pass
+auth        sufficient                                   pam_winbind.so {if "with-krb5":krb5_auth} {if "with-krb5-cache":krb5_ccache_type=KEYRING} {if "with-winbind-cache":cached_login} use_first_pass
```

Comment:
This needs to be solved differently as the new default cache is KCM and not KEYRING anymore! So you need to allow to set the type via the commandline option.